src/apis/drive/samples/.eslintrc.yml

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
So this introduces a problem.  Today when we run the generator, we `rm -rf src/apis`, and then generate everything from scratch.  Adding samples into the same directory as the generated code will require a more sophisticated generator that's capable of making updates in place instead of just blasting everything away.  I'm not opposed to that approach, it just needs to be done.

Reply:
We can put samples elsewhere (or leave them in their current location). I'll see what we can do in the generator though.

Reply:
OK - after some thinking, I think it will be better to leave samples in the separate folder, at least because of `sampleClient.js` that is shared across all APIs. I suggest that all samples will have `package.json` requiring the corresponding API, just as we do in GAPIC libraries.

Reply:
I actually prefer the other approach 😆 We need to talk this one through in person, because it plays into the whole "how do we auto-generate individual library changes" discussion.  This likely means some stuff with Kokoro.